package.json

```diff
@@ -242,7 +242,7 @@
 			{
 				"name": "Resource usage",
 				"gridItemConfig": {
-				  "sizex": 3,
+				  "sizex": 4,
```

Comment:
Increased this to 4 because above, Properties is increased to 2, to align upper and bottom half